specification/web/resource-manager/Microsoft.Web/2016-03-01/Provider.json

```diff
@@ -92,7 +92,7 @@
           "200": {
             "description": "OK",
             "schema": {
-              "$ref": "#/definitions/Object"
+              "type": "object"
```

Comment:
an SDK breaking change.

Reply:
Change from #/definitions/Object is SDK but recommended by Azure SDK/ARM